frontend/src/pages/About/About.jsx

```diff
@@ -20,6 +20,26 @@ const useStyles = makeStyles(() => ({
 
 export const About = (props) => {
   let dark = props.theme;
+  const [team, setTeam] = useState([]);
+  useEffect(() => {
+    getTeam();
+  }, []);
+  const getTeam = async () => {
+    try {
+      const url = `${END_POINT}/teamMember/getTeamMembers/`;
+      const response = await fetch(url);
+      const data = await response.json();
+      const _data = data.map((item) => {
+        return {
+          ...item,
+          teams: item.teams[0].split(","),
+        };
+      });
+      setTeam(_data);
+    } catch (error) {
```

Comment:
can we please a generic toast message for successful fetch/error?

Reply:
Done